packages/tooltip/src/initTooltips.tsx

```diff
@@ -21,11 +21,13 @@ const forEachElement = (selector: string, callback: Function) => {
  * Hydrates tooltip for SSR pages
  */
 const initTooltips = () => {
-  forEachElement('[data-tooltip-from-article-converter]', (el: HTMLElement) => {
```

Comment:
Denne blei vel lagt til for å skille mellom tooltips fra ndla-frontend og article-converter. No vil vel alle tooltips hydreres. Er det problematisk?